ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/dhcp-servers.md

```diff
@@ -0,0 +1,92 @@
+# dhcp-servers
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+- [DHCP Server](#dhcp-server)
+  - [DHCP Servers Summary](#dhcp-servers-summary)
+  - [DHCP Server Subnets](#dhcp-server-subnets)
+  - [DHCP Server IPv4 Vendor Options](#dhcp-server-ipv4-vendor-options)
+  - [DHCP Server Configuration](#dhcp-server-configuration)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+## DHCP Server
+
+### DHCP Servers Summary
+
+| DHCP Server Enabled | VRF | IPv4 DNS Domain | IPv6 DNS Domain |
+| ------------------- | --- | --------------- | --------------- |
+| True | - | - | - |
+| True | TEST | testv4.com | testv6.com |
+
+### DHCP Server Subnets
+
+| Subnet | VRF |
+| ------ | --- |
+| 2a00:2::/64 | - |
+| 10.0.0.0/24 | TEST |
+| 10.2.3.0/24 | - |
+
+### DHCP Server IPv4 Vendor Options
+
+| Vendor ID | Sub-option Code | Sub-option Type | Sub-option Data |
+| --------- | ----------------| --------------- | --------------- |
+| NTP | 1 | string | test |
+| NTP | 42 | ipv4-address | 10.1.1.1 |
```

Comment:
I feel like these options should be per server

Reply:
Have implemented per VRF since i though that was the best/only identifier to arrange by.